requirements.txt

```diff
@@ -1,5 +1,5 @@
 django-cursor-pagination
-pulpcore>=3.6
+pulpcore>=3.6,<3.9
```

Comment:
should not it be >=3.7,<3.9

Reply:
The plugin is still compatible with 3.6, there were no incompatible changes for it in pulpcore 3.7 I believe.